discord/ext/commands/view.py

```diff
@@ -108,21 +108,50 @@ def __repr__(self):
 
 # Parser
 
+# map from opening quotes to closing quotes
+quotes = {
```

Comment:
This needs to be a private variable, e.g. `_quotes`. There is no need to expose this.